scripts/get-link-json.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+# Report errors for links to Red Hat JIRA bugs that are behind a login
+
+
+
+# Get Github Token
+# Use local token for now
+
+REPO_OWNER="rohennes" # OR should it be "openshift" - the curl request only works with rohennes as it's a forked repo. The API accepts merged sha ID, not PR commit ID?
+REPO_NAME="openshift-docs"
+
+# Get repo name - hardcode to openshift-docs?
+
+# Commit ID
+COMMIT_ID=$(git log -n 1 --pretty=format:"%H")
+echo $COMMIT_ID
+# Get pull id
+PULL_NUMBER=$(curl -s "https://api.github.com/repos/$REPO_OWNER/openshift-docs/commits/$COMMIT_ID/pulls"  | jq '.[0].number')
```

Comment:
This seems to work OK:

commit_id=$(git log -n 1 --pretty=format:"%H")
pr_number="$(curl -s "https://api.github.com/search/issues?q=$commit_id" | jq '.items[0].number')"